openapi/src/main/scala/endpoints/openapi/DocumentedRequests.scala

```diff
@@ -0,0 +1,41 @@
+package endpoints
+package openapi
+
+/**
+  * Interpreter for [[algebra.DocumentedRequests]].
+  */
+trait DocumentedRequests
+  extends algebra.DocumentedRequests
+    with DocumentedUrls
+    with Methods {
+
+  type RequestHeaders[A] = DocumentedHeaders
+
+  case class DocumentedHeaders(value: List[String])
+
+  def emptyHeaders = DocumentedHeaders(Nil)
+
+  type Request[A] = DocumentedRequest
+
+  case class DocumentedRequest(
+    method: Method,
+    url: DocumentedUrl,
+    headers: DocumentedHeaders,
+    entity: Option[DocumentedRequestEntity]
+  )
+
+  type RequestEntity[A] = Option[DocumentedRequestEntity]
+
+  case class DocumentedRequestEntity(description: Option[String], content: Map[String, MediaType])
```

Comment:
Can we have `String` in `content` either aliased or commented(to know what it expresses without looking at usages)?